src/user/UserConsents.tsx

```diff
@@ -65,7 +65,7 @@ export const UserConsents = () => {
     messageKey: t("users:revokeClientScopes", {
       clientId: selectedClient?.clientId,
     }),
-    continueButtonLabel: "common:delete",
+    continueButtonLabel: "common:Revoke",
```

Comment:
I believe this should start with a lower case.
```suggestion
    continueButtonLabel: "common:revoke",
```

Reply:
When I change it to lower case it appears as lower case in the button. revoke doesn't seem to be effected by "common"
I looked around for a few min to see if I could but wasn't able to do so. 

![image](https://user-images.githubusercontent.com/31955648/180430656-58610079-c0b8-4442-b83a-13802538a6d4.png)

Reply:
I think this is because this key does not exist in any of the language bundles, so instead it defaults to the name of the key. You'll have to add a `revoke` entry to [`public/resources/en/common.json`](https://github.com/keycloak/keycloak-admin-ui/blob/main/public/resources/en/common.json).